cmds/core/insmod/tinygo_tag_noasm_amd64.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017-2018 the u-root Authors. All rights reserved
```

Comment:
just checking, is there a test for this? I do have a test, badbuild, which is for things that should break a build, if you want to test it there.

Reply:
There isn't. I was wondering however if we should add tests to compile everything that should be tinygo compilable to prevent regression there? Or aren't we there yet? Given the rapid progress of things it seems like it might be a good time for it.
Anyway, right now, I don't think there are any tests that use tinygo. I noticed that tinygo -dev builds are available now via nightlies.

Reply:
@leongross To build my tinygo busybox, I
(1) used makebb to build just {init,shutdown,gosh} as "go-bb" and put it in a pre-symlinked directory
(2) modified mkuimage so that it would honor the symlinks above
(3) built u-root with u-root/gobusybox#121 passing appropriate --init and --sh options (to point to go-bb)